mlir/utils/jenkins/MIOpenDriver.py

```diff
@@ -30,7 +30,7 @@
 # Compiled regexp object used for extracting elapsed time from MIOpenDriver's output
 ELAPSED_TIME_RE = re.compile(r"Elapsed: (.*)ms")
 
-class Paths(NamedTuple):
```

Comment:
I think the NamedTuple usage makes sense here, why remove it?

Reply:
Restored NamedTuple.